subprojects/packagefiles/libsndfile/meson.build

```diff
@@ -104,11 +104,12 @@ sndfile = configure_file(
 
 incdirs = include_directories('.', 'src')
 
+pymod = import('python')
 def = custom_target(
   'def',
   output: 'sndfile.def',
   capture: true,
-  command: [ './src/create_symbols_file.py', 'win32', meson.project_version() ],
+  command: [ pymod.find_installation('python3'), meson.project_source_root() / 'src/create_symbols_file.py', 'win32', meson.project_version() ],
```

Comment:
`pymod.find_installation('python3')` returns the same thing as `find_program('python3')`, except for the additional methods useful for building CPython extensions and installing python software to site-packages.

Also importing pymod is a mild performance penalty.